AS/src/main/java/org/example/Classify.java

```diff
@@ -0,0 +1,9 @@
+package org.example;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Set;
+
+public interface Classify {
```

Comment:
I think is better to change to **Classifier**. Generally, interfaces should be named adjectives or nouns.